src/libvirtApi/domain.ts

```diff
@@ -1124,6 +1129,36 @@ export function domainResume({
     return call(connectionName, objPath, 'org.libvirt.Domain', 'Resume', [], { timeout, type: '' });
 }
 
+export function domainSuspendToDisk({
```

Comment:
I think in the code we should call these things by their libvirt names, like `domainManagedSave`. Only the UI will say "Suspend to disk".